implementations/tutorial_state_preparation.py

```diff
@@ -95,14 +95,10 @@ def layer(params, j):
 
 ##############################################################################
 # To set up the device, we select a plugin that is compatible with
-# evaluating expectations through sampling: the ``forest.qvm`` plugin. The
-# syntax is slightly different than for other plugins, we need to also
-# feed a ``device`` keyword specifying the number of qubits in the format
-# ``[number of qubits]q-pyqvm``. The keyword ``shots`` indicates the
-# number of samples used to estimate expectation values.
-#
+# evaluating expectations through sampling: the ``qiskit.aer`` plugin. The keyword ``shots``
+# indicates the number of samples used to estimate expectation values.
 
-dev = qml.device("forest.qvm", device="3q-pyqvm", shots=1000)
+dev = qml.device('qiskit.aer', wires=3, shots=1000)
```

Comment:
```suggestion
dev = qml.device('qiskit.aer', backend='qasm_simulator', wires=3, shots=1000)
```

Best to make this explicit so that we don't get questions!

Reply:
My thought was that explicitly specifying a backend raises the question of _why_ we specified that backend, whereas if we leave it to the default then the question of why the default goes to the makers of PL-Qiskit.